app/models/host/base.rb

```diff
@@ -196,6 +196,13 @@ def set_interfaces(parser)
       if !self.managed? && self.primary_interface.mac.blank? && self.primary_interface.identifier.blank?
         identifier, values = parser.suggested_primary_interface(self)
         self.primary_interface.mac = Net::Validations.normalize_mac(values[:macaddress]) if values.present?
```

Comment:
on this line, there's already `if values.present?`, it can probably be moved to the block below and the condition could be removed